### DISCUSSION TITLE
Download whole playlist

### DISCUSSION THREAD

**Discussion by RiccardoBarbieri** [KRTirtho/spotube#1069] [Q&A] [answered]
Is it possible to download whole playlists instead of single tracks? I can't seem to find the option.

**Comment by nepalivai** [comment, marked as answer]
yes, it is possible. Here are the steps,
1. go to the play list you want to download.
2. click on titles.<img width="960" alt="image" src="https://github.com/KRTirtho/spotube/assets/108126089/dd5abad1-1a9f-47a9-806a-9f86a3e3b264">
3. More actions <img width="763" alt="image" src="https://github.com/KRTirtho/spotube/assets/108126089/54f3a1e3-381d-4d47-9c62-f2f2ff351356">
4. Download. <img width="780" alt="image" src="https://github.com/KRTirtho/spotube/assets/108126089/ba548347-346a-4828-aaa6-36ed953787f2">

## note
<img width="537" alt="image" src="https://github.com/KRTirtho/spotube/assets/108126089/d2ca2af7-cb9e-4c1a-9344-6ac222338a91">

**Reply by RiccardoBarbieri** [reply]
Thank you, I didn't notice the options to select all titles.

**Reply by nepalivai** [reply]
So you got your answer, now how do you mark it as an answer?

**Reply by Tenuka22** [reply]
> Thank you, I didn't notice the options to select all titles.



> yes, it is possible. Here are the steps,
> 
> 1. go to the play list you want to download.
> 2. click on titles.<img alt="image" width="960" src="https://private-user-images.githubusercontent.com/108126089/298449868-dd5abad1-1a9f-47a9-806a-9f86a3e3b264.png?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3MzI5NTcyOTksIm5iZiI6MTczMjk1Njk5OSwicGF0aCI6Ii8xMDgxMjYwODkvMjk4NDQ5ODY4LWRkNWFiYWQxLTFhOWYtNDdhOS04MDZhLTlmODZhM2UzYjI2NC5wbmc_WC1BbXotQWxnb3JpdGhtPUFXUzQtSE1BQy1TSEEyNTYmWC1BbXotQ3JlZGVudGlhbD1BS0lBVkNPRFlMU0E1M1BRSzRaQSUyRjIwMjQxMTMwJTJGdXMtZWFzdC0xJTJGczMlMkZhd3M0X3JlcXVlc3QmWC1BbXotRGF0ZT0yMDI0MTEzMFQwODU2MzlaJlgtQW16LUV4cGlyZXM9MzAwJlgtQW16LVNpZ25hdHVyZT1hODNkMTc0NjhjNjEzNjlkYmE5MzQ2MjRmN2M3YmMxMDRjNjk4YjhkYWVlODBmOTIwODE5NDVmNGFkNGMxOWRkJlgtQW16LVNpZ25lZEhlYWRlcnM9aG9zdCJ9.OyX5SFaRixWvI0OWNve9osnLsE_xukLUsmWG3rZZIp8">
> 3. More actions <img alt="image" width="763" src="https://private-user-images.githubusercontent.com/108126089/298450023-54f3a1e3-381d-4d47-9c62-f2f2ff351356.png?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3MzI5NTcyOTksIm5iZiI6MTczMjk1Njk5OSwicGF0aCI6Ii8xMDgxMjYwODkvMjk4NDUwMDIzLTU0ZjNhMWUzLTM4MWQtNGQ0Ny05YzYyLWYyZjJmZjM1MTM1Ni5wbmc_WC1BbXotQWxnb3JpdGhtPUFXUzQtSE1BQy1TSEEyNTYmWC1BbXotQ3JlZGVudGlhbD1BS0lBVkNPRFlMU0E1M1BRSzRaQSUyRjIwMjQxMTMwJTJGdXMtZWFzdC0xJTJGczMlMkZhd3M0X3JlcXVlc3QmWC1BbXotRGF0ZT0yMDI0MTEzMFQwODU2MzlaJlgtQW16LUV4cGlyZXM9MzAwJlgtQW16LVNpZ25hdHVyZT1lOTIxZWQ5MmJmMWE5MjdhZWI0ZjllYzNjYTlmYjIxOGFlYWU1ZTcyZmY5OGQ0N2NkYzA5OTIwNGE2YjA2YTIzJlgtQW16LVNpZ25lZEhlYWRlcnM9aG9zdCJ9.uan3agRWHCm0bcF8uYZsyZuiQ35XhCqkYiz3dWWm_Iw">
> 4. Download. <img alt="image" width="780" src="https://private-user-images.githubusercontent.com/108126089/298449974-ba548347-346a-4828-aaa6-36ed953787f2.png?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3MzI5NTcyOTksIm5iZiI6MTczMjk1Njk5OSwicGF0aCI6Ii8xMDgxMjYwODkvMjk4NDQ5OTc0LWJhNTQ4MzQ3LTM0NmEtNDgyOC1hYWE2LTM2ZWQ5NTM3ODdmMi5wbmc_WC1BbXotQWxnb3JpdGhtPUFXUzQtSE1BQy1TSEEyNTYmWC1BbXotQ3JlZGVudGlhbD1BS0lBVkNPRFlMU0E1M1BRSzRaQSUyRjIwMjQxMTMwJTJGdXMtZWFzdC0xJTJGczMlMkZhd3M0X3JlcXVlc3QmWC1BbXotRGF0ZT0yMDI0MTEzMFQwODU2MzlaJlgtQW16LUV4cGlyZXM9MzAwJlgtQW16LVNpZ25hdHVyZT1hZDc0NDA5NmQ1ZmE2MGM2ZmFlM2EwZjhmOTQwYzQ0OTFkMWU4YmY5NzhmOTU2YmZhZTU1Y2I4MWE5ZTM5MzZkJlgtQW16LVNpZ25lZEhlYWRlcnM9aG9zdCJ9.u6rKpKfoMSqj0mkTowS_t4xNfENF3aoz012V9zg7NlQ">
> 
> ## note
> <img alt="image" width="537" src="https://private-user-images.githubusercontent.com/108126089/298450190-d2ca2af7-cb9e-4c1a-9344-6ac222338a91.png?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3MzI5NTcyOTksIm5iZiI6MTczMjk1Njk5OSwicGF0aCI6Ii8xMDgxMjYwODkvMjk4NDUwMTkwLWQyY2EyYWY3LWNiOWUtNGMxYS05MzQ0LTZhYzIyMjMzOGE5MS5wbmc_WC1BbXotQWxnb3JpdGhtPUFXUzQtSE1BQy1TSEEyNTYmWC1BbXotQ3JlZGVudGlhbD1BS0lBVkNPRFlMU0E1M1BRSzRaQSUyRjIwMjQxMTMwJTJGdXMtZWFzdC0xJTJGczMlMkZhd3M0X3JlcXVlc3QmWC1BbXotRGF0ZT0yMDI0MTEzMFQwODU2MzlaJlgtQW16LUV4cGlyZXM9MzAwJlgtQW16LVNpZ25hdHVyZT1iNGQ0MDVmM2ZiMzVkZTBjNmMzM2EzMDU5NjBlY2I2YjRiMmQxMTExNWYzYTQzMTMxODBmZmFhZDI3ZjI2N2ZiJlgtQW16LVNpZ25lZEhlYWRlcnM9aG9zdCJ9.ql09MGL9eABL2z-8p0iZ8sASjnCgFRW1U449MoLIY64">



How to enable auto download for all the music i play, like cache while playing

**Reply by KRTirtho** [reply]
We're planningg to add caching support. Follow #127 discussion
